js/views/userPageVw.js

```diff
@@ -505,7 +504,7 @@ module.exports = Backbone.View.extend({
 
     $.ajax({
       type: "DELETE",
-      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'),
+      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'), //TODO: Change from GET to POST for safe methods, http://restcookbook.com/HTTP%20Methods/idempotency/
```

Comment:
What does this refer to?

The gets in the line with the TODO are Backbone get calls to the item model.